<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Live data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="ctime">
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Collimation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Detector position:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="coll15">
        <property name="text">
         <string>15m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="coll10">
        <property name="text">
         <string>10m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="coll20">
        <property name="text">
         <string>20m</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="detpos">
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Count time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="listLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Past files:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="fileList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="widgetLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/print</normaloff>:/print</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Close window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUnzoom">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/viewfull</normaloff>:/viewfull</iconset>
   </property>
   <property name="text">
    <string>&amp;Unzoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as PDF...</string>
   </property>
  </action>
  <action name="actionLogScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Logarithmic Y scale</string>
   </property>
   <property name="iconText">
    <string>Log Scale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionNormalized">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Normalized</string>
   </property>
   <property name="iconText">
    <string>Normalized</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L&amp;egend</string>
   </property>
  </action>
  <action name="actionResetPlot">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionWriteXml">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>Save as XML...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSetAsROI">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/find</normaloff>:/find</iconset>
   </property>
   <property name="text">
    <string>Set as ROI</string>
   </property>
  </action>
  <action name="actionSelectChannels">
   <property name="text">
    <string>Select channels...</string>
   </property>
  </action>
  <action name="actionLoadTOF">
   <property name="text">
    <string>Load TOF...</string>
   </property>
  </action>
  <action name="actionLoadPAD">
   <property name="text">
    <string>Load PAD...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCustomRange">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom range</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
